cypress/integration/e2e/management-tab/edit-worker.spec.ts

```diff
@@ -1,54 +1,118 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { HoursInfoCells } from "../../../support/commands";
 describe("Tab management", () => {
   beforeEach(() => {
     cy.loadScheduleToMonth();
     cy.get('[data-cy="btn-management-tab"]').click();
     cy.get('[data-cy="management-page-title"]').should("be.visible");
-    cy.get('[data-cy="btn-add-worker"]').click();
-    cy.get('[data-cy="worker-drawer"]').should("be.visible");
   });
 
-  describe("Creating worker", () => {
-    context("when creating valid worker", () => {
-      const newWorker = "Ala makota";
+  describe("Worker drawer management", () => {
+    beforeEach(() => {
+      cy.get('[data-cy="btn-add-worker"]').click();
+      cy.get('[data-cy="worker-drawer"]').should("be.visible");
+    });
+
+    describe("Creating worker", () => {
+      context("when creating valid worker", () => {
+        const newWorker = "Ala makota";
+
+        it("creates the worker", () => {
+          cy.get('[data-cy="name"]').type(newWorker);
+          cy.get(`[value="${newWorker}"]`).should("be.visible");
+          cy.get('[data-cy="position"]').click().get('[data-cy="other"]').click();
+          cy.get('[data-cy="contract"]').click().get('[data-cy="employment_contract"]').click();
+          cy.get('[data-cy="contract-time-dropdown"]').click().get('[data-cy="other"]').click();
+          cy.get('[data-cy="input-employ-time-other"]').click().type("{backspace}{backspace}13");
+          cy.get('[data-cy="btn-save-worker"]').click();
+          cy.get('[data-cy="btn-add-worker"]').should("be.visible");
+          cy.contains(newWorker);
+          cy.get('[data-cy="btn-schedule-tab"]').click();
+          cy.contains(newWorker);
+        });
+      });
+    });
+
+    describe("Editing the time", () => {
+      beforeEach(() => {
+        cy.get('[data-cy="contract"]').click();
+      });
 
-      it("creates the worker", () => {
-        cy.get('[data-cy="name"]').type(newWorker);
-        cy.get(`[value="${newWorker}"]`).should("be.visible");
-        cy.get('[data-cy="position"]').click().get('[data-cy="other"]').click();
-        cy.get('[data-cy="contract"]').click().get('[data-cy="employment_contract"]').click();
+      it("properly handles employment contract", () => {
+        cy.get('[data-cy="employment_contract"]').click();
+        cy.get('[data-cy="contract-time-dropdown"]').click().get('[data-cy="full"]').click();
```

Comment:
te zmiany bez zapisania ich są nieistotne

Reply:
dobra, widzę z przeniesione z innego miejsca, ale nadal można to usunąć

Reply:
Czyli co, usunąć ten test cały?